### PR TITLE
add travis support with industrial_ci

### PR DESCRIPTION
**Pull request by JeroenDM**
Use same file as https://github.com/Jmeyer1292/opw_kinematics

### CODE REVIEW

**Comment by JeroenDM** [comment]
@simonschmeisser feel free to ignore this pull request and create your own. I'm an absolute novice when it comes to travis building :)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+git:
+  quiet: true
+
+env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
+    - ROS_REPO=ros
+  matrix:
+    - ROS_DISTRO="kinetic"
```
Comment (simonschmeisser) [inline .travis.yml]:
```suggestion
    - ROS_DISTRO="kinetic" UPSTREAM_WORKSPACE=file
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - ROS_DISTRO="melodic"
```
Comment (simonschmeisser) [inline .travis.yml]:
```suggestion
    - ROS_DISTRO="melodic" UPSTREAM_WORKSPACE=file
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+install:
+  - git clone --quiet --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - git clone https://github.com/ros-industrial/kuka_experimental
```
Comment (simonschmeisser) [inline .travis.yml]:
```suggestion

```
remove all of these

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  - git clone https://github.com/JeroenDM/kuka_kr6r700_moveit_config
```
Comment (simonschmeisser) [inline .travis.yml]:
```suggestion

```

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  - git clone https://github.com/ros-planning/moveit_resources
```
Comment (simonschmeisser) [inline .travis.yml]:
```suggestion

```

**Comment by simonschmeisser** [comment]
https://github.com/ros-industrial/industrial_ci/blob/legacy/doc/index.rst#use-rosinstall-file-to-specify-the-depended-packages-source-repository

add a file called `.travis.rosinstall` containing a rosinstall file with the dependencies (and probably the actual code as well?)

**Comment by simonschmeisser** [comment]
both `industrial_ci` and `moveit_ci` use some prepared nested docker images where the actual testing is done so you need to pass that in to the script. I actually never tried that however :)

**Comment by JeroenDM** [comment]
Hooray!

**Comment by JeroenDM** [comment]
> add a file called `.travis.rosinstall` containing a rosinstall file with the dependencies (and probably the actual code as well?)

I'm not sure what you mean with "the actual code as well".

Is the current setup correct? @simonschmeisser

**Comment by JeroenDM** [comment]
There are probably some other changes necessary when the kinetic and melodic branches diverge.

**Comment by simonschmeisser** [comment]
That looks very good to me and should be merged! (after squashing or using squash.on merge)

I wasn't sure if you need to add this repo in the rosinstall or if this is done automatically (which it is as we now know, so just ignore that comment) :)

**Review by simonschmeisser — APPROVED** [review]

**Pull request merged**